.eslintrc.js

```diff
@@ -54,19 +68,41 @@ module.exports = {
             ignoreRestSiblings: true,
```

Comment:
We don't need this rule modification anymore, that landed in our ESLint config in v10